include/moq/detail/messages.h

```diff
@@ -24,6 +25,7 @@ namespace moq::messages {
     using SubscribeId = uint64_t;
     using TrackAlias = uint64_t;
     using ParamType = uint64_t;
+    using Extensions = std::map<uint64_t, std::vector<uint8_t>>;
```

Comment:
It doesn't really matter. Since it is likely never going to be that large it's more space efficient to use map, and the lookup times are going to be inconsequential.

Reply:
Ok to merge, but map won't work for standard interop, since keys can repeat with different values, so we need something like multimap, or a simple array if order becomes important in the standard. Order is currently unspecified, but likely to be mandated eventually since E2EE/AEAD ciphers require exact order.